### PR TITLE
Added Dutch translation (#2535)

### PR DESCRIPTION
**Pull request by geppyz**
Signed-off-by: Gerben D <geppyz@gmail.com>

### CODE REVIEW

File: ui/main/src/assets/i18n/nl.json [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+      },
+      "type": {
+        "title": "Type"
+      },
+      "showCardsWithResponse": {
+        "label": "Kaarten met reactie van mijn entiteit"
+      },
+      "ack": {
+        "title": "Erkenning",
+        "acknowledged": {
```
Comment (JeroenGommans) [inline ui/main/src/assets/i18n/nl.json:123]: Bevestigen ?

File: ui/main/src/assets/i18n/nl.json [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+      "unread": "Ongelezen en dan datum"
+    },
+    "tips": {
+      "acknowledged": "Erkend",
+      "lttd": "Resterende tijd om te beslissen",
+      "hasAResponseFromYourEntity" : "Uw entiteit heeft al gereageerd",
+      "loadingInProgress": "Kaart wordt geladen"
+    },
+    "acknowledgeAllCards": {
+      "label": "Erken alle kaarten",
```
Comment (JeroenGommans) [inline ui/main/src/assets/i18n/nl.json:150]: Bevestig

**Review by JeroenGommans — COMMENTED** [review]
Great. Maybe chnage  "acknowledgement" from "erknning" to "bevestiging"?

**Comment by geppyz** [comment]
Build failure seems to be a fluke, did a manual re-run: https://github.com/opfab/operatorfabric-core/actions/runs/1824718383

Was perfectly fine.

**Review by JeroenGommans — APPROVED** [review]

**Pull request merged**